common/scala/src/main/scala/whisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -69,7 +70,7 @@ class KafkaConsumerConnector(
     val wakeUpTask = actorSystem.scheduler.scheduleOnce(cfg.sessionTimeoutMs.milliseconds + 1.second)(consumer.wakeup())
 
     try {
-      val response = consumer.poll(duration.toMillis).asScala
+      val response = consumer.poll(duration).asScala
```

Comment:
Is wakeup still needed with the new poll method?

Reply:
Yes I believe the need for wakeup still applies here. At the very least, `poll` is documented to still throw wakeup exceptions.